tests/io_performance/image/run-test.sh

```diff
@@ -21,33 +21,48 @@ cp config.fio ~/"$FOLDER_GIT"/Configs/
 echo "Running IOSTAT $result"
 ./run-iostat.sh &
 
-# Running FIO
-echo "Running FIO"
-fio config.fio --output-format=normal,json > ~/"$FOLDER_GIT"/Configs/Test-results/fio-results
-
-echo "Result FIO generate start"
-./fioconv ~/"$FOLDER_GIT"/Configs/Test-results/fio-results ~/"$FOLDER_GIT"/SUMMARY.csv
-echo "Result FIO generate done"
-
-# Git configurate
-echo "Started configuring GitHub"
-git config --global http.sslverify false
-(cd ~ && git clone https://"$GIT_LOGIN":"$GIT_TOKEN"@github.com/"$GIT_LOGIN"/"$GIT_REPO")
-git config --global user.email "fio_test@example.com"
-git config --global user.name "FIO"
-echo "GitHub configuration is done"
-
-if [ -z "$GIT_PATH" ]
-then mv ~/"$FOLDER_GIT"/ ~/"$GIT_REPO"/
-else mv ~/"$FOLDER_GIT"/ ~/"$GIT_REPO"/"$GIT_PATH"/
-fi
+if [ -z "$GIT_REPO" ] || [ -z "$GIT_LOGIN" ] || [ -z "$GIT_TOKEN" ]
+then
+    # Running FIO
+    echo "Running FIO without publish results on github.com"
+    fio config.fio --output-format=normal
+    echo "FIO tests are end"
+else
+    # Running FIO
+    echo "Running FIO"
+    fio config.fio --output-format=normal,json > ~/"$FOLDER_GIT"/Configs/Test-results/fio-results
 
-# Create a new folder in the GIT repository and push the changes
-echo "Create a folder and start posting results to GIT"
-if [ -z "$GIT_BRANCH" ]
-then (cd ~/"$GIT_REPO"/ && git add ~/"$GIT_REPO"/"$FOLDER_GIT" && git commit -m "fio-results $FOLDER_GIT" && git push)
-else (cd ~/"$GIT_REPO"/ && git checkout -b "$GIT_BRANCH" && git add ~/"$GIT_REPO"/"$FOLDER_GIT" && git commit -m "io-results $FOLDER_GIT" && git push --set-upstream origin "$GIT_BRANCH")
-fi
-echo "FIO tests are end: $FOLDER_GIT"
+    echo "Result FIO generate start"
+    ./fioconv ~/"$FOLDER_GIT"/Configs/Test-results/fio-results ~/"$FOLDER_GIT"/SUMMARY.csv
+    echo "Result FIO generate done"
+
+    # Git configurate
+    echo "Started configuring GitHub"
+    git config --global http.sslverify false
+    (cd ~ && git clone https://"$GIT_LOGIN":"$GIT_TOKEN"@github.com/"$GIT_LOGIN"/"$GIT_REPO")
```

Comment:
Can you please take a look onto duplicate of `$GIT_LOGIN`? I think, we need to remove the second one and expect something like `itmo-eve/eve-performance` inside `$GIT_REPO`. WDYT?

Reply:
Yes, I agree, your suggestion makes sense if you need to upload results to your organization's account.
Otherwise, there was nothing wrong or not logical in the double, except in the case of the organization.
Updated.

Reply:
Thank you!